### PR TITLE
add note about EFS and Preview Pane

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5100

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/information-protection/windows-information-protection/limitations-with-wip.md [hunk 1]
```diff
@@ -134,6 +134,9 @@ This table provides info about the most common problems you might encounter whil
     </tr>
 </table>
 
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/limitations-with-wip.md]:
```suggestion
> [!NOTE]
```

File: windows/security/information-protection/windows-information-protection/limitations-with-wip.md [hunk 2]
```diff
@@ -134,6 +134,9 @@ This table provides info about the most common problems you might encounter whil
...
+>When corporate data is written to disk, WIP uses the Windows-provided Encrypting File System (EFS) to protect it and associate it with your enterprise identity. One caveat to keep in mind is that the Preview Pane in File Explorer will not work for encrypted files.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/limitations-with-wip.md]:
```suggestion
> When corporate data is written to disk, WIP uses the Windows-provided Encrypting File System (EFS) to protect it and associate it with your enterprise identity. One caveat to keep in mind is that the Preview Pane in File Explorer will not work for encrypted files. 
```

File: windows/security/information-protection/windows-information-protection/limitations-with-wip.md [hunk 3]
```diff
@@ -134,6 +134,9 @@ This table provides info about the most common problems you might encounter whil
     </tr>
 </table>
 
+>[!NOTE]
+>When corporate data is written to disk, WIP uses the Windows-provided Encrypting File System (EFS) to protect it and associate it with your enterprise identity. One caveat to keep in mind is that the Preview Pane in File Explorer will not work for encrypted files. 
+
 >[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/limitations-with-wip.md]:
```suggestion
> [!NOTE]
```

File: windows/security/information-protection/windows-information-protection/limitations-with-wip.md [hunk 4]
```diff
@@ -134,6 +134,9 @@ This table provides info about the most common problems you might encounter whil
...
 >Help to make this topic better by providing us with edits, additions, and feedback. For info about how to contribute to this topic, see [Contributing to our content](https://github.com/Microsoft/windows-itpro-docs/blob/master/CONTRIBUTING.md).
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/windows-information-protection/limitations-with-wip.md]:
```suggestion
> Help to make this topic better by providing us with edits, additions, and feedback. For info about how to contribute to this topic, see [Contributing to our content](https://github.com/Microsoft/windows-itpro-docs/blob/master/CONTRIBUTING.md).       
```

**Pull request merged**